AppKit/CoreAnimation/CPAnimationContext.j

```diff
@@ -263,10 +268,8 @@ CPLog.debug(_cmd + "context stack =" + _CPAnimationContextStack);
 - (void)getAnimations:(CPArray)cssAnimations getTimers:(CPArray)timers forView:(CPView)aTargetView usingAction:(Object)anAction rootView:(CPView)rootView cssAnimate:(BOOL)needsCSSAnimation
 {
     var keyPath = anAction.keypath,
-        isFrameKeyPath = (keyPath == @"frame" || keyPath == @"frameSize"),
-        customLayout = [aTargetView hasCustomLayoutSubviews],
-        customDrawing = [aTargetView hasCustomDrawRect],
-        needsFrameTimer = isFrameKeyPath && (customLayout || customDrawing);
+        isFrameKeyPath  = (keyPath === @"frame" || keyPath === @"frameSize" || keyPath === @"frameOrigin"),
+        needsFrameTimer = isFrameKeyPath && [[aTargetView animator] wantPeriodicFrameUpdates];
```

Comment:
Can you rename needsFrameTimer to needsPeriodicFrameUpdates because it's no longer a timer. Thanks.